Readme.md

```diff
@@ -3,61 +3,67 @@
 
 ```cs
 @Html.DevExpress().GridView(settings => {
-    ...
+    // ...
     settings.CallbackRouteValues = new { Controller = "EFGridMode", Action = "GridViewPartial" };
 }).Bind(Model).GetHtml()
 ```
 
-<p> </p><p>- The <a href="http://documentation.devexpress.com/#AspNet/CustomDocument14760"><u>Database Server mode</u></a>. All data shaping operations are performed on the database server side. This mode is effective when it is necessary to maintain a large volume of data only, for example 100k records. Since this mode creates special LINQ queries, this mode may be not so effective when operating with small portion of data:</p>
-
 ```cs
 NorthwindContext db = new NorthwindContext();
-```
 
-<p> </p>
-
-```cs
 public ActionResult GridViewPartial() {
-    return PartialView(db.Orders);
+    return PartialView(db.Orders.ToList());
 }
 ```
 
-<p> </p>
+## Database server mode
+
+In database server mode, all data shaping operations are performed on the database server side. This mode is effective when it is necessary to maintain a large amount of data, for instance 100k records. The GridView loads records on demand and performs data-aware operations (sorting, filtering, grouping, etc.) on the data server. This approach significantly improves the GridView’s speed and responsiveness.
+
+
 
+Call a [BindToEF](https://docs.devexpress.com/AspNetMvc/DevExpress.Web.Mvc.GridViewExtension.BindToEF.overloads) method to bind the grid to the Entity Framework data content. This method uses database server mode to optimize the execution of all queries to the data context initiated by the GridView. 
 ```cs
 @Html.DevExpress().GridView(settings => {
-    ...
+    // ...
     settings.CallbackRouteValues = new { Controller = "EFDatabaseServerMode", Action = "GridViewPartial" };
 }).BindToEF(string.Empty, string.Empty, (s, e) => {
     e.KeyExpression = "OrderID";
     e.QueryableSource = Model;
 }).GetHtml()
 ```
 
-<p> </p><p>This example operates the <strong>Northwind SQL Compact demo database</strong>. The data Model is defined via the <strong>EF Code First</strong> approach. Since tables’ column names contain spaces, it is necessary to define the corresponding mappings at the <strong>DBContext</strong> level (via the <strong>OnModelCreating</strong> method). In most cases, this step is no longer required.</p>
+```cs
+NorthwindContext db = new NorthwindContext();
+
+public ActionResult GridViewPartial() {
+    return PartialView(db.Orders);
+}
+```
+
+This example operates the `Northwind SQL Compact` demo database data Model. Since table column names contain spaces, it is necessary to define the corresponding mappings at the `DBContext` level (see the [OnModelCreating](https://github.com/LanaDX/how-to-bind-the-gridview-with-the-entity-framework-in-a-regular-and-database-server-modes-e3252/blob/0fc02fe03420146b1d0138c8038cb7167b5521a3/CS/Models/Model.cs#L18-L23) method implementation). In most cases, this step is not required.
```

Comment:
What do you mean when you say `This example operates`?

Reply:
```suggestion
In this example, the grid is bound to the `Northwind SQL Compact` demo database data Model. Since table column names contain spaces, it is necessary to define the corresponding mappings at the `DBContext` level (see the [OnModelCreating](https://github.com/LanaDX/how-to-bind-the-gridview-with-the-entity-framework-in-a-regular-and-database-server-modes-e3252/blob/0fc02fe03420146b1d0138c8038cb7167b5521a3/CS/Models/Model.cs#L18-L23) method implementation). In most cases, this step is not required.
```

Reply:
Ok